src/test/ui/consts/const-external-macro-const-err.rs

```diff
@@ -0,0 +1,13 @@
+// edition:2018
+// aux-build:external_macro.rs
+
+// Ensure that CONST_ERR lint errors
+// are not silenced in external macros.
+// https://github.com/rust-lang/rust/issues/65300
+
+extern crate external_macro;
+use external_macro::static_assert;
+
+fn main() {
+    static_assert!(2 + 2 == 5); //~ ERROR
```

Comment:
This works outside "expression" scope too, right? Can we test it there? I seem to recall some differences, but maybe that was a transient state...

Reply:
As in the following?
```rust
static_assert!(false); //~ ERROR
fn main() { /* ... */ }
```